oscarlab/graphene
# Meson: Pal/Linux-SGX

## Description of the changes 

Mesonification of Linux-SGX.

SGX' `aesm.proto` was moved to a common directory for two reasons: it will be merged with python's `aesm.proto` and `protoc-c` is capricious about paths.

## How to test this PR? 

Check compilation. Check different SGX drivers.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/2411)
